src/main/java/mate/academy/intro/repository/BookSpecificationBuilder.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.intro.repository;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.intro.dto.BookSearchParametersDto;
+import mate.academy.intro.model.Book;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
+public class BookSpecificationBuilder implements SpecificationBuilder<Book> {
+    private static final String TITLE_COLUMN = "title";
+    private static final String AUTHOR_COLUMN = "author";
+    private static final String ISBN_COLUMN = "isbn";
```

Comment:
better make these constants public and reuse them. it is better to have one place for future changes